examples/variational_inference/bayesian_neural_network_advi.ipynb

```diff
@@ -190,7 +190,7 @@
    },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.            ann_input = pm.Data("ann_input", minibatch_x, mutable=True, dims=("obs_id", "train_cols"))
</pre><p>mutability is True by default </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/719/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Changing this would also change a bunch of stuff in the diff -- we'd have to re-execute the notebook, otherwise "Check cells were executed sequentially" would fail during `pre-commit`. Since this PR is just running the `pre-commit` hook for someone else's (seemingly-abandoned) PR, I'd prefer to just leave it if it's fine.